Squirrel/Program.cs

```diff
@@ -1,27 +1,51 @@
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.UI.Services;
 using Microsoft.AspNetCore.Localization;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Options;
 using Squirrel;
 using Squirrel.Contexts;
+using Squirrel.Entities;
+using Squirrel.Services;
 using System.Globalization;
 
 var builder = WebApplication.CreateBuilder(args);
 var services = builder.Services;
 var configuration = builder.Configuration;
 
+services.AddTransient<IEmailSender, EmailSender>();
+
+services.AddSingleton(configuration.GetSection("EmailConfiguration").Get<EmailConfiguration>());
```

Comment:
It's better to carry `"EmailConfiguration"` out to a static class with constants